actors/miner/tests/miner_actor_test_commitment.rs

```diff
@@ -161,44 +133,6 @@ mod miner_actor_test_commitment {
         h.check_state(&rt);
     }
 
-    #[test]
-    fn deal_space_exceeds_sector_space() {
```

Comment:
Potentially worth introducing this in proveCommitAggregate test.  Or ConfirmSectorProofsValid though that seems a bit too annoying to be worth it.